packages/modules/auth/src/models/auth-identity.ts

```diff
@@ -29,15 +29,9 @@ export default class AuthIdentity {
   @Property({ columnType: "text" })
   provider: string
 
-  @Property({ columnType: "text" })
-  scope: string
-
   @Property({ columnType: "jsonb", nullable: true })
   user_metadata: Record<string, unknown> | null
 
-  @Property({ columnType: "jsonb" })
-  app_metadata: Record<string, unknown> = {}
```

Comment:
question: I thought we wanted to keep `app_metadata` for other auth-specific props?

Reply:
seb was in favour to remove it as we have the provider data

Reply:
Okay I might have missed that discussion, so I don't think I have the context to review the PR. Would just be a bunch of questions 😅

Reply:
@olivermrbl that's fine, I can answer all the questions you have, it's important we get this right :)